### ISSUE TITLE
feat: support clearing history and turning off ability to save it

### ISSUE THREAD

**Issue by mds1** [ScopeLift/umbra-protocol#471]
> Can be a different PR, but we should probably also include a button to erase the history, and possibly a feature that let's the user turn off history saving completely

Let's do this one after #465 is merged

**Comment by alexkeating** [comment]
Going to split turning off ability to save into a separate issue
